### PR TITLE
Backend languages

### PR DESCRIPTION
**Pull request by nathan-stytch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
 1. Copy the .env_template to your own .env file (`cp .env_template .env`)
 1. Add your Stytch API credentials from the API keys tab in your Stytch dashboard. You'll need to update `STYTCH_PROJECT_ID` and `STYTCH_SECRET` in your `.env` file with your project ID and secret from the dashboard.
 1. From the API keys tab, create a public token and use it for `STYTCH_PUBLIC_TOKEN` in `Login.js`.
-1. Run the app: `npm start`
-1. Open the app in your browser at `localhost:9000`
 
-# Session management
+## Build the client
+
+2. Build the front end files from the client directory. Note: You'll need to rebuild your files to reflect any changes you want to make.
```
Comment (mary-stytch) [inline README.md:15]:
```suggestion
1. Build the frontend files from the client directory. Note: You'll need to rebuild your files to reflect any changes you want to make.
```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+   ```
+   $ cd client
+   $ npm run build
+   ```
+3. This example app serves the frontend when you run a backend server. Click on the links below Supported Languages to your preferred language to see instructions on how to run the server.
```
Comment (mary-stytch) [inline README.md:20]: We should always use 1 for numbering so we don't need to keep track of the numbers.
```suggestion
1. This example app serves the frontend when you run a backend server. Click on the links under Supported server languages to go to the readme for your preferred language.
```

File: README.md [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
+
+## Supported server languages
 
-This app sets a variable in React state for authenticating a user to illustrate the user experience for authenticating with Stytch - we don't recommend doing this! You should use a more secure session management solution, like [express-session](https://www.npmjs.com/package/express-session).
+Click on the language to navigate to the setup instructions for each server
+[Python (3.7)](https://github.com/stytchauth/stytchjs-react-express/tree/main/server/python)
+[node](https://github.com/stytchauth/stytchjs-react-express/tree/main/server/node)
```
Comment (mary-stytch) [inline README.md:26]:
```suggestion
[Node](https://github.com/stytchauth/stytchjs-react-express/tree/main/server/node)
```

File: client/src/App.css [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (mary-stytch) [inline client/src/App.css:1]:
```suggestion
```

File: client/src/Login.js [hunk 5]
```diff
@@ -2,40 +2,44 @@ import { Stytch } from "@stytch/stytch-react";
 import React from "react";
 
 const Login = () => {
-  const STYTCH_PUBLIC_TOKEN = "public-token-1111-1111-1111";
+  const STYTCH_PUBLIC_TOKEN =
+    "public-token-test-196bf55b-4d4d-4694-bb9c-4312a2398cf6";
```
Comment (mary-stytch) [inline client/src/Login.js:6]: Please revert this change

File: client/src/Login.js [hunk 6]
```diff
@@ -2,40 +2,44 @@ import { Stytch } from "@stytch/stytch-react";
...
   const stytchProps = {
     config: {
       loginConfig: {
-        magicLinkUrl: "http://localhost:9000/authenticate",
+        magicLinkUrl: `http://localhost:3000/authenticate`,
```
Comment (mary-stytch) [inline client/src/Login.js:10]: Please use a different port than 3000

File: server/node/README.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Node backend server
+
+Set up an Express server that will serve the front end and make API calls to Stytch.
```
Comment (mary-stytch) [inline server/node/README.md:3]: Can you please also add a link to the client instructions?
```suggestion
This directory contains an example Express server that authenticates your magic link via the Stytch API. It also serves the frontend React app
```

File: server/python/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
+# Python (3.7) backend server
+
+Set up an Express server that will serve the front end and make API calls to Stytch.
```
Comment (mary-stytch) [inline server/python/README.md:3]: Please link the frontend readme here as well.
```suggestion
This directory contains example python code to run a Flask server that authenticates your magic link with the stytch-python client library. It also serves the frontend React app
```

File: server/python/server.py [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+@app.route("/", methods=["GET"], defaults={"path": None})
+@app.route("/<path:path>", methods=["GET"])
+def index(path):
+    return send_from_directory(app.static_folder, "index.html")
+
+
+if __name__ == "__main__":
+    app.run()
+
+# TODO: Integrate flask-login
```
Comment (mary-stytch) [inline server/python/server.py:97]: Is this TODO intentional?

File: client/src/Login.js (hunk 5 above)
Comment (nathan-stytch) [reply client/src/Login.js]: sorry, forgot to revert changes to this file

fixed: https://github.com/stytchauth/stytchjs-react-express/pull/14/commits/0f346ca34b958a86d4b8c1abcade734d0a37d7d1

File: server/python/server.py (hunk 9 above)
Comment (nathan-stytch) [reply server/python/server.py:97]: yes, i would like to leave it in here

Comment (mary-stytch) [reply server/python/server.py:97]: cool just making sure

File: client/src/Login.js (hunk 6 above)
Comment (nathan-stytch) [reply client/src/Login.js]: fixed: https://github.com/stytchauth/stytchjs-react-express/commit/0f346ca34b958a86d4b8c1abcade734d0a37d7d1

**Review by mary-stytch — APPROVED** [review]
Looks good
